tekton_configs/test-run.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-testing
+  namespace: gabrielwen
+spec:
+  pipelineSpec:
+   tasks:
+   - name: xgboost-synthetic
+     taskRef:
+      name: pytest
+     params:
+     - name: src-dir
+       value: /mnt/testdir/src/gabrielwen
+     - name: repos
+       value: kubeflow/examples@HEAD,kubeflow/testing@HEAD,kubeflow/tf-operator@HEAD
+     - name: python-path
+       value: /mnt/testdir/src/gabrielwen/kubeflow/examples/py:/mnt/testdir/src/gabrielwen/kubeflow/testing/py:/mnt/testdir/src/gabrielwen/kubeflow/tf-operator/py
+     - name: get-cred-args-array
+       value:
+       - "-m"
+       - "kubeflow.testing.get_kf_testing_cluster"
+       - "get-credentials"
+     - name: pytest-args-array
+       value:
+        - xgboost_test.py
+        - --log-cli-level=info
+        - --log-cli-format='%(levelname)s|%(asctime)s|%(pathname)s|%(lineno)d| %(message)s'
+        - --timeout=1800
+        - --junitxml=/mnt/testdir/output/gabrielwen/junit_xgboost-synthetic-test.xml
+        - --notebook_artifacts_dir=/mnt/testdir/output/gabrielwen/xgboost-synthetic-test-notebooks
+        - --repos=kubeflow/examples@HEAD,kubeflow/tf-operator@HEAD
```

Comment:
It looks like the test currently launches a K8s job which will use init containers to pull down the repos as well as install papermill.

Should we add a step to this task that builds a container that bakes in the source code and papermill? Then the job we launch just needs to run papermill.

Reply:
yea I think we definitely need to update part of our codes for the new convention, we can discuss this later.